CRM/Contribute/Form/Task/Status.php

```diff
@@ -310,6 +310,8 @@ public static function processForm($form, $params) {
    * @return array
    */
   public static function &getDetails($contributionIDs) {
+    //Fix for empty param, avoiding payment hangs
+    $contributionIDs = (!empty($contributionIDs))? $contributionIDs : 'NULL' ;
```

Comment:
@mfrausto-mdw i'm wondering if instead of NULL it should be 1 so the query would be like (1) @eileenmcnaughton @monishdeb